docs/lessons/dance-party.md

```diff
@@ -0,0 +1,511 @@
+
+Next we need to set the tile map. Drag a ``||scene:set tile to with wall||`` block from ``||scene:scene||`` and drag it into ``||loops:on start||``. Next click on the grey color oval and select white. Then go into the editor and take a bucket of paint with white selected and make the entire tile white. Switch wall to on. 
+
+Now we need to actually create the tile map. To do this go into ``||scene:scene||`` once again and grab a ``||scene:set tile map to||`` block and drag it under ``||scene:set tile to with wall||``. Go into the map editor and take the same color we used for setting the tile and draw a line at the bottom of the screen. Now we have a "wall" at the bottom of the screen. 
+
+To make the background less plain you can add scene effects. To do this go into ``||scene:scene||`` and pull a ``||scene:start screen effect||`` block. Go to the drop down and select star field to create an outer space theme. 
+
+```blocks
+scene.setBackgroundColor(15)
+scene.setTile(1, img`
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+    1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
+`, true)
+scene.setTileMap(img`
+    . . . . . . . . . .
+    . . . . . . . . . .
+    . . . . . . . . . .
+    . . . . . . . . . .
+    . . . . . . . . . .
+    . . . . . . . . . .
+    . . . . . . . . . .
+    1 1 1 1 1 1 1 1 1 1
+`)
+effects.starField.startScreenEffect()
+```
+
+Congratulations! You have now set the scene for your game.
+
+## Part Two: Creating the Sprite
+
+To set the sprite grab a ``||sprites:set mySprite to sprite of kind player||`` from ``||sprites:sprites||`` and drag it into ``||loops:on start||``. Now click on mySprite and select rename variable and rename your sprite to whatever you would like to call your player. In the ``||sprites:sprite of kind player||`` block, select the grey square and draw an image of your player. 
+
+Now we want to set our sprites starting position. To do this go into sprites and grab ``||sprites:set mySprite position to x y||`` and drag it under ``||sprites:sprite of kind player||``. Now set the x position to **80** and the y position to **100**. 
+
+```blocks
+let steve: Sprite = sprites.create(img`
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . f f f . . . . . . .
+    . . . . . . f f f . . . . . . .
+    . . . . . . f f f . . . . . . .
```

Comment:
Maybe set a different color for the background in the first step? the head blends in fully with the background as is and looks like steve is headless.

Other benefit to that is that setting a background color will make your lesson stand out even more on the homepage (a lot of the tutorials / etc have black backgrounds, so it's nice to break that up whenever possible)